evergreen/config.yml

```diff
@@ -922,7 +955,7 @@ tasks:
   commands:
   - func: "launch remote baas"
     vars:
-      baas_branch: master
+      baas_branch: 3f31617aacfe5d31b9057fc298b735b60acd6424
```

Comment:
is this so James can update the scripts and tests first and then we can use master?

Reply:
I imagine other changes will require newer baas version so i'd be good to work on pretty much latest commit.